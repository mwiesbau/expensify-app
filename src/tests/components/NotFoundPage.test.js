import React from 'react'
import { shallow } from 'enzyme'
import NotFoundPage from '../../components/NotFoundPage'

test("Not found snapshot page", () => {
  const wrapper = shallow(<NotFoundPage />)
  expect(wrapper).toMatchSnapshot()
})