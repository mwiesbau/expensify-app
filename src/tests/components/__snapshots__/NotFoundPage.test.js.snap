// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found snapshot page 1`] = `
<p>
  404! - 
  <Link
    to="/"
  >
    Go home
  </Link>
</p>
`;
